knora/dsplib/models/helpers.py

```diff
@@ -242,8 +248,6 @@ def add_context(self, prefix: str, iri: Optional[str] = None) -> None:
                 return
             if prefix in self.common_ontologies:
                 self._context[prefix] = self.common_ontologies[prefix]
-            else:
-                raise BaseError("The prefix '{}' is not known!".format(prefix))
```

Comment:
If I understand correctly, you basically did 3 things in helpers.py:
 1. tidy up without semantic changes
 2. remove error raisings
 3. add more external ontos

Like Balduin, I have troubles understanding the code, but I just wanted to ask if you are sure that you want to remove all the errors that were raised.

Reply:
1. correct
2. I just removed one error because we now allow all ontologies (it is not checked anymore if it is known or not)
3. correct

The point of still having some ontology IRIs is that when using `dsp-tools get` the IRIs of the external ontologies can be converted back to the notation with the prefix. If the ontology is not known, the fully qualified IRI is used.